docs/en/get_started.md

```diff
@@ -118,11 +118,11 @@ Take the latest precompiled package as example, you can install it as follows:
 
 ```shell
 # install MMDeploy
-wget https://github.com/open-mmlab/mmdeploy/releases/download/v0.7.0/mmdeploy-0.7.0-linux-x86_64-onnxruntime1.8.1.tar.gz
-tar -zxvf mmdeploy-0.7.0-linux-x86_64-onnxruntime1.8.1.tar.gz
-cd mmdeploy-0.7.0-linux-x86_64-onnxruntime1.8.1
-pip install dist/mmdeploy-0.7.0-py3-none-linux_x86_64.whl
-pip install sdk/python/mmdeploy_python-0.7.0-cp38-none-linux_x86_64.whl
+wget https://github.com/open-mmlab/mmdeploy/releases/download/v0.8.0/mmdeploy-0.8.0-linux-x86_64-onnxruntime1.8.1.tar.gz
```

Comment:
Did this means we have to update the version after every release?

Reply:
I am afraid so because the instruction is related to mmdeploy version. I haven't figured out a better way. I think we can discuss about it later on.